polygon/heimdall/event_fetch_test.go

```diff
@@ -0,0 +1,39 @@
+package heimdall
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/erigontech/erigon-lib/log/v3"
+)
+
+func TestOver50EventBlockFetch(t *testing.T) {
```

Comment:
this test should maybe be removed from main since it is calling an external service `https://heimdall-api.polygon.technology/` and may fail our CI